core/src/main/java/cz/xtf/core/image/Product.java

```diff
@@ -29,6 +29,22 @@ public String templatesBranch() {
         return resolveDefaultingProperty("templates.branch");
     }
 
+    public String helmChartsRepo() {
+        return resolveDefaultingProperty("helm.charts.repo");
+    }
+
+    public String helmChartsBranch() {
+        return resolveDefaultingProperty("helm.charts.branch");
+    }
+
+    public String helmChartsLocation() {
```

Comment:
It's just parametrized directory within repository for helm charts (e.g https://github.com/wildfly/wildfly-charts/tree/main/charts). This just provides option to parameterized the location for charts.

Reply:
So is this property required? If I do not specify `helm.charts.location` will it default to `charts` ?

Reply:
You're correct that the property is optional, however the defaulting behaviour is managed by test suites (or other software using XTF), in our use case, it will default to `charts`

Reply:
Ok, I agree default in xtf based on EAP knowledge could miss in other products. Not sure there is some convention among different products how helm repo is structured. I believe not.

Setting default in testsuite seems correct to me.